packages/server/lib/exception.coffee

```diff
@@ -48,7 +48,9 @@ module.exports = {
       user and user.authToken
 
   create: (err) ->
-    return Promise.resolve() if process.env["CYPRESS_ENV"] isnt "production"
+    if process.env["CYPRESS_ENV"] isnt "production" or
```

Comment:
You can use Promise.method here http://bluebirdjs.com/docs/api/promise.method.html